### DISCUSSION TITLE
How to import django-idom into a django project?

### DISCUSSION THREAD

**Discussion by legacy-chase** [idom-team/idom#839] [Question] [answered]
The docs don't specify the correct to way to import Django-IDOM into a django project.

I have added 'django_idom' to INSTALLED_APPS, as is typical for django add ons, but I receive the following error when running manage.py runserver:

```
File "/Users/xxx/Projects/nuthatch/venv/lib/python3.11/site-packages/django/template/defaulttags.py", line 1027, in find_library
    return parser.libraries[name]
           ~~~~~~~~~~~~~~~~^^^^^^
KeyError: 'idom'
```

I have also tried adding 'idom' to INSTALLED_APPS. In this case runserver starts, but I receive a 500 error from django:

```
TemplateSyntaxError at /

'idom' is not a registered tag library. Must be one of:
...
```

Does anyone know the right way?

**Comment by rmorshea** [comment, marked as answer]
Have you checked out the documentation [here](https://idom-team.github.io/django-idom/getting-started/installation/)?

**Reply by Archmonger** [reply]
@legacy-chase To clarify, the `Django-IDOM` docs are separate from the `IDOM` documentation.

As Ryan mentioned, see the following link: https://idom-team.github.io/django-idom/getting-started/installation/
